<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>522</width>
    <height>546</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>522</width>
    <height>546</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>522</width>
    <height>546</height>
   </size>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QGridLayout" name="gridLayout_3" rowstretch="0,0" rowminimumheight="200,0">
      <item row="1" column="0">
       <layout class="QGridLayout" name="gridLayout">
        <property name="horizontalSpacing">
         <number>2</number>
        </property>
        <property name="verticalSpacing">
         <number>0</number>
        </property>
        <item row="1" column="3">
         <widget class="QPushButton" name="N3">
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>50</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>20</pointsize>
            <kerning>true</kerning>
           </font>
          </property>
          <property name="text">
           <string>3</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="N2">
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>50</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>20</pointsize>
            <kerning>true</kerning>
           </font>
          </property>
          <property name="text">
           <string>2</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="N1">
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>50</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>20</pointsize>
            <kerning>true</kerning>
           </font>
          </property>
          <property name="text">
           <string>1</string>
          </property>
         </widget>
        </item>
        <item row="3" column="3">
         <widget class="QPushButton" name="N6">
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>50</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>20</pointsize>
            <kerning>true</kerning>
           </font>
          </property>
          <property name="text">
           <string>6</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QPushButton" name="N4">
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>50</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>20</pointsize>
            <kerning>true</kerning>
           </font>
          </property>
          <property name="text">
           <string>4</string>
          </property>
         </widget>
        </item>
        <item row="5" column="3">
         <widget class="QPushButton" name="blank2">
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>50</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>20</pointsize>
            <kerning>true</kerning>
           </font>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QPushButton" name="N0">
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>50</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>20</pointsize>
            <kerning>true</kerning>
           </font>
          </property>
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item row="4" column="3">
         <widget class="QPushButton" name="N9">
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>50</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>20</pointsize>
            <kerning>true</kerning>
           </font>
          </property>
          <property name="text">
           <string>9</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QPushButton" name="N8">
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>50</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>20</pointsize>
            <kerning>true</kerning>
           </font>
          </property>
          <property name="text">
           <string>8</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QPushButton" name="blank1">
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>50</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>20</pointsize>
            <kerning>true</kerning>
           </font>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QPushButton" name="N5">
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>50</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>20</pointsize>
            <kerning>true</kerning>
           </font>
          </property>
          <property name="text">
           <string>5</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QPushButton" name="N7">
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>50</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>20</pointsize>
            <kerning>true</kerning>
           </font>
          </property>
          <property name="text">
           <string>7</string>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <spacer name="verticalSpacer_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="0" column="0" colspan="3">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QFrame" name="display">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(186, 186, 186);</string>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <widget class="QStackedWidget" name="stackedWidget">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>501</width>
             <height>201</height>
            </rect>
           </property>
           <property name="currentIndex">
            <number>1</number>
           </property>
           <widget class="QWidget" name="page">
            <widget class="QLabel" name="insertCardLabel">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>501</width>
               <height>201</height>
              </rect>
             </property>
             <property name="font">
              <font>
               <pointsize>21</pointsize>
              </font>
             </property>
             <property name="text">
              <string>INSERT CARD</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </widget>
           <widget class="QWidget" name="page_2">
            <widget class="QLabel" name="pinCodeLabel">
             <property name="geometry">
              <rect>
               <x>160</x>
               <y>60</y>
               <width>181</width>
               <height>29</height>
              </rect>
             </property>
             <property name="styleSheet">
              <string notr="true">color: rgb(0, 0, 0);
font: 16pt &quot;Segoe UI&quot;;</string>
             </property>
             <property name="text">
              <string>Enter pincode</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
            <widget class="QLineEdit" name="pinCodeLineEdit">
             <property name="geometry">
              <rect>
               <x>170</x>
               <y>90</y>
               <width>161</width>
               <height>44</height>
              </rect>
             </property>
             <property name="font">
              <font>
               <pointsize>20</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">border-color: rgb(255,255,255);
border-width: 2px;
border-style: solid;
border-radius: 9px;
padding: 2px;</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
            <widget class="QLabel" name="infoLabel">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>150</y>
               <width>501</width>
               <height>31</height>
              </rect>
             </property>
             <property name="font">
              <font>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </widget>
           <widget class="QWidget" name="page_3">
            <widget class="QPushButton" name="pushButton_2">
             <property name="geometry">
              <rect>
               <x>420</x>
               <y>80</y>
               <width>80</width>
               <height>51</height>
              </rect>
             </property>
             <property name="styleSheet">
              <string notr="true">color: rgb(0, 0, 0);</string>
             </property>
             <property name="text">
              <string>Saldo</string>
             </property>
            </widget>
            <widget class="QPushButton" name="pushButton_3">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>140</y>
               <width>80</width>
               <height>51</height>
              </rect>
             </property>
             <property name="styleSheet">
              <string notr="true">color: rgb(0, 0, 0);</string>
             </property>
             <property name="text">
              <string>Takaisin</string>
             </property>
            </widget>
            <widget class="QPushButton" name="withdrawButton">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>80</y>
               <width>80</width>
               <height>51</height>
              </rect>
             </property>
             <property name="styleSheet">
              <string notr="true">color: rgb(0, 0, 0);</string>
             </property>
             <property name="text">
              <string>Nosta</string>
             </property>
            </widget>
            <widget class="QPushButton" name="pushButton_5">
             <property name="geometry">
              <rect>
               <x>420</x>
               <y>140</y>
               <width>80</width>
               <height>51</height>
              </rect>
             </property>
             <property name="styleSheet">
              <string notr="true">color: black</string>
             </property>
             <property name="text">
              <string>Tilitapahtumat</string>
             </property>
            </widget>
            <widget class="QLabel" name="label">
             <property name="geometry">
              <rect>
               <x>180</x>
               <y>50</y>
               <width>131</width>
               <height>41</height>
              </rect>
             </property>
             <property name="font">
              <font>
               <pointsize>14</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">color:rgb(0, 0, 0)
</string>
             </property>
             <property name="text">
              <string>Valitse toiminto</string>
             </property>
            </widget>
           </widget>
           <widget class="QWidget" name="page_4">
            <widget class="QLabel" name="label_2">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>70</y>
               <width>501</width>
               <height>41</height>
              </rect>
             </property>
             <property name="font">
              <font>
               <pointsize>14</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">color:rgb(0, 0, 0)
</string>
             </property>
             <property name="text">
              <string>Poista kortti</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
            <widget class="QLabel" name="labelPoistaKorttiInfo">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>29</y>
               <width>501</width>
               <height>31</height>
              </rect>
             </property>
             <property name="font">
              <font>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="text">
              <string>TextLabel</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </widget>
           <widget class="QWidget" name="page_5">
            <widget class="QPushButton" name="debitButton">
             <property name="geometry">
              <rect>
               <x>30</x>
               <y>120</y>
               <width>101</width>
               <height>51</height>
              </rect>
             </property>
             <property name="styleSheet">
              <string notr="true">color:rgb(0, 0, 0);
font: 10pt &quot;Segoe UI&quot;;</string>
             </property>
             <property name="text">
              <string>Debit</string>
             </property>
            </widget>
            <widget class="QPushButton" name="creditButton">
             <property name="geometry">
              <rect>
               <x>200</x>
               <y>120</y>
               <width>101</width>
               <height>51</height>
              </rect>
             </property>
             <property name="styleSheet">
              <string notr="true">color:rgb(0, 0, 0);
font: 10pt &quot;Segoe UI&quot;;</string>
             </property>
             <property name="text">
              <string>Credit</string>
             </property>
            </widget>
            <widget class="QPushButton" name="sharedAccountButton">
             <property name="geometry">
              <rect>
               <x>370</x>
               <y>120</y>
               <width>101</width>
               <height>51</height>
              </rect>
             </property>
             <property name="styleSheet">
              <string notr="true">color:rgb(0, 0, 0);
font: 10pt &quot;Segoe UI&quot;;</string>
             </property>
             <property name="text">
              <string>Shared account</string>
             </property>
            </widget>
            <widget class="QLabel" name="label_3">
             <property name="geometry">
              <rect>
               <x>180</x>
               <y>40</y>
               <width>151</width>
               <height>41</height>
              </rect>
             </property>
             <property name="styleSheet">
              <string notr="true">color:rgb(0, 0, 0);
font: 12pt &quot;Segoe UI&quot;;</string>
             </property>
             <property name="text">
              <string>Choose Account</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </widget>
           <widget class="QWidget" name="page_6">
            <widget class="QPushButton" name="sharedBackButton">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>150</y>
               <width>111</width>
               <height>41</height>
              </rect>
             </property>
             <property name="styleSheet">
              <string notr="true">color:rgb(0, 0, 0);
font: 10pt &quot;Segoe UI&quot;;</string>
             </property>
             <property name="text">
              <string>Back</string>
             </property>
            </widget>
            <widget class="QListWidget" name="listSharedAccountWidget">
             <property name="geometry">
              <rect>
               <x>120</x>
               <y>0</y>
               <width>261</width>
               <height>201</height>
              </rect>
             </property>
             <item>
              <property name="text">
               <string>1 , Kalle Perkele, credit</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>1 , Kalle Perkele, credit</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>1 , Kalle Perkele, credit</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>1 , Kalle Perkele, credit</string>
              </property>
             </item>
            </widget>
           </widget>
          </widget>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="1" colspan="2">
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QPushButton" name="okButton">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>50</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Segoe UI</family>
            <pointsize>12</pointsize>
            <italic>false</italic>
            <bold>false</bold>
            <kerning>true</kerning>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(34, 255, 0);
font: 12pt &quot;Segoe UI&quot;;</string>
          </property>
          <property name="text">
           <string>ENTER</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="clearButton">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>50</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Segoe UI</family>
            <pointsize>12</pointsize>
            <italic>false</italic>
            <bold>false</bold>
            <kerning>true</kerning>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 0);
font: 12pt &quot;Segoe UI&quot;;</string>
          </property>
          <property name="text">
           <string>CLEAR</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="cancelButton">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>50</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Segoe UI</family>
            <pointsize>12</pointsize>
            <italic>false</italic>
            <bold>false</bold>
            <kerning>true</kerning>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 0, 4);
font: 12pt &quot;Segoe UI&quot;;</string>
          </property>
          <property name="text">
           <string>CANCEL</string>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="insertCardButton">
          <property name="minimumSize">
           <size>
            <width>240</width>
            <height>50</height>
           </size>
          </property>
          <property name="text">
           <string>Korttiluukku</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
